### PR TITLE
Update Raft option text, setup guide text

### PR DESCRIPTION
**Pull request by SunnyBat**
## What is this fixing or adding?
This updates the Raft Setup Guide to include running RML as administrator as a potential solution for when RML does not load properly. This also updates the description of some options in Raft to be more specific about what they do.

## How was this tested?
Loaded website locally, verified it worked.

## If this makes graphical changes, please attach screenshots.
WebhostScreenshot

### CODE REVIEW

File: worlds/raft/Options.py [hunk 1]
```diff
@@ -53,7 +60,7 @@ class IslandGenerationDistance(Choice):
     default = 8
 
 class ExpensiveResearch(Toggle):
-    """Makes unlocking items in the Crafting Table consume the researched items."""
+    """If No is selected, researching items and unlocking items in the Crafting Table works the same as vanilla Raft. If Yes is selected, each unlock in the Crafting Table will require its own set of researched items in order to unlock it."""
```
Comment (Exempt-Medic) [inline worlds/raft/Options.py:63]: You might want to split this out into two lines

File: worlds/raft/Options.py [hunk 2]
```diff
@@ -66,8 +73,7 @@ class BigIslandEarlyCrafting(Toggle):
     display_name = "Early recipes behind big islands"
 
 class PaddleboardMode(Toggle):
-    """Sets later story islands to in logic without an Engine or Steering Wheel. May require lots of paddling. Not
-    recommended."""
+    """Sets later story islands to in logic without an Engine or Steering Wheel. May require lots of paddling."""
```
Comment (Exempt-Medic) [inline worlds/raft/Options.py:76]:
```suggestion
    """Sets later story islands to be in logic without an Engine or Steering Wheel. May require lots of paddling."""
```

File: worlds/raft/Options.py [hunk 3]
```diff
@@ -32,7 +32,13 @@ class FillerItemTypes(Choice):
     option_both = 2
 
 class IslandFrequencyLocations(Choice):
-    """Sets where frequencies for story islands are located."""
+    """Sets where frequencies for story islands are located.
+    Vanilla will keep frequencies in their vanilla, non-randomized locations.
+    Random on Island will randomize each frequency within its vanilla island, but will preserve island order.
```
Comment (Exempt-Medic) [inline worlds/raft/Options.py:37]:
```suggestion
    Random On Island will randomize each frequency within its vanilla island, but will preserve island order.
```

File: worlds/raft/Options.py [hunk 4]
```diff
@@ -32,7 +32,13 @@ class FillerItemTypes(Choice):
...
+    Random Island Order will change the order you visit islands, but will preserve the vanilla location of each frequency unlock.
+    Random On Island Random Order will randomize the location containing the frequency on each island and randomize the order.
+    Progressive will randomize the frequencies to anywhere, but will always unlock the frequencies in vanilla order as the frequency items are received.
```
Comment (Exempt-Medic) [inline worlds/raft/Options.py:40]:
```suggestion
    Progressive will randomize the frequencies to anywhere, but will always unlock the frequencies in the vanilla order as the frequency items are received.
```

Comment (SunnyBat) [reply worlds/raft/Options.py:40]: I think this change is unnecessary, as "vanilla order" is already referencing the frequencies. Adding "their" does not change this or make it any clearer that the frequencies are what are being referenced, nor that it's the vanilla order of the frequencies.

Comment (Exempt-Medic) [reply worlds/raft/Options.py:40]: Fair enough

**Review by Exempt-Medic — APPROVED** [review]
LGTM, comments were addressed.

**Review by agilbert1412 — APPROVED** [review]
Looks good!

**Comment by NewSoupVi** [comment]
Note: This PR was opened before the big options page PR changed option tooltip rendering.

However, that change looks to be getting reverted https://github.com/ArchipelagoMW/Archipelago/pull/3513

I'm waiting for the final verdict on that PR to merge this

**Comment by SunnyBat** [comment]
@NewSoupVi Looks like the PR got merged. I just checked this by merging into `main` and it looks fine to me. This PR is fine to merge.

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**